src/ripper/ripcddialog.cpp

```diff
@@ -343,21 +351,13 @@ void RipCDDialog::DeviceSelected(int device_index) {
           SLOT(SongsLoaded(SongList)));
 }
 
-void RipCDDialog::Finished(Ripper* ripper) {
+void RipCDDialog::Finished(Ripper& ripper, float progress_to_display) {
```

Comment:
Please follow the style guide and only use const references or pointers for non-primitive parameters. https://google.github.io/styleguide/cppguide.html

Reply:
The guide states

> Non-optional input parameters should usually be values or const references, while **non-optional** output and **input/output parameters should usually be references** (which cannot be null). 

so that's what I did here. I can't make it const since the callback must clean up the instance using the `deleteLater` function, which is not const.

Reply:
They've changed the style guide since I last read it :-)

In the interest of maintaining a consistent style throughout the codebase I'd still prefer it if this were changed to `Ripper*`. We don't use non-const references anywhere else.

Reply:
Done :)